src/utils/editor/source-editor.js

```diff
@@ -70,7 +89,7 @@ class SourceEditor {
    * bottom.
    * @memberof utils/source-editor
    */
-  alignLine(line, align = "top") {
+  alignLine(line: number, align: string = "top") {
```

Comment:
I'm not familiar with `alignLine`.  It seems it can be called in [source-search.js](https://github.com/devtools-html/debugger.html/blob/master/src/utils/editor/source-search.js#L174) and [Editor/index.js](https://github.com/devtools-html/debugger.html/blob/master/src/components/Editor/index.js#L384). `align` seems to be type `string` in both cases. I could be totally wrong though...  Please let me know. Thank you!